packages/strapi-design-system/src/LiveRegions/useNotifyAT.ts

```diff
@@ -2,24 +2,24 @@ import { useEffect } from 'react';
 
 import { LiveRegionIds } from './constants';
 
-const notifyLog = (message) => {
-  const logEl = document.querySelector(`#${LiveRegionIds.Log}`);
+const notifyLog = (message: string) => {
+  const logEl = document.querySelector(`#${LiveRegionIds.Log}`) as HTMLElement;
```

Comment:
I don't think `as HTMLElement` is necessary since we check if the variable exists anyways (same for the rest of the file)

Reply:
It is and it isnt, `querySelector` returns `Element` so `innerText` does not exist on `Element`, but you can pass a generic to `querySelector` so i'll move to that.